_inc/client/at-a-glance/scan.jsx

```diff
@@ -236,6 +238,13 @@ class DashScan extends Component {
 		return false;
 	}
 
+	getUpgradeContent() {
+		return renderCard( {
+			className: 'jp-dash-item__is-inactive',
```

Comment:
I know this follows an established pattern, but I wanted to say it's always a better idea to create these as modifier classes, for consistency, predictability and readability. So in that case, we'd have `.is-inactive` as a class and it would be used as a modifier here: `jp-dash-item is-inactive`.

Definitely no need to change this in this PR, just wanted to point it out.